src/EthereumNamingService.ts

```diff
@@ -27,11 +27,11 @@ export abstract class EthereumNamingService extends NamingService {
   };
 
   static readonly UrlMap: BlockhanNetworkUrlMap = {
-    mainnet: 'https://mainnet.infura.io',
-    ropsten: 'https://ropsten.infura.io',
-    kovan: 'https://kovan.infura.io',
-    rinkeby: 'https://rinkeby.infura.io',
-    goerli: 'https://goerli.infura.io',
+    mainnet: 'https://main-rpc.linkpool.io',
+    ropsten: 'https://ropsten-rpc.linkpool.io',
+    rinkeby: undefined, 
+    goerli: undefined, 
+    kovan: undefined, 
```

Comment:
We can just omit them. I don't see why we should highlight that we don't support some testnets, while there are a lot more of them

Reply:
ok